source/_integrations/energyid.markdown

```diff
@@ -0,0 +1,161 @@
+related:
+  - docs: /integrations/energy
+    title: Energy management documentation
+  - url: https://www.energyid.eu/
+    title: EnergyID Platform
+---
+
+The **EnergyID** {% term integration %} enables seamless transfer of energy metrics from Home Assistant to [EnergyID](https://www.energyid.eu/), a professional energy management platform for long-term tracking and analysis.
+
+## About EnergyID
+
+EnergyID provides tools for:
+
+- Historical energy consumption analysis
+- Solar production tracking across multiple sites
+- Utility bill management and validation
+- Carbon footprint reporting
+- Multi-property energy comparisons
+
+## Prerequisites
+
+You will need:
+
+1. Active EnergyID account with webhook permissions
+2. Webhook URL from EnergyID's integration settings
+3. Sensor entity in Home Assistant with valid numerical readings
+4. Configured metric type in EnergyID that matches your sensor data
+
+## How you can use this integration
+
+- **Long-term trend analysis**: Track energy patterns over years
+- **Multi-site management**: Aggregate data from multiple properties
+- **etc**
+{% include integrations/config_flow.md %}
+
+## Configuration options
+
+{% configuration_basic %}
+EnergyID Webhook URL:
+  description: "Unique URL provided by EnergyID's webhook configuration (required)"
+Home Assistant Entity ID:
+  description: "Sensor entity to monitor for state changes (required)"
+EnergyID Metric:
+  description: "Measurement category from EnergyID's supported metrics (required)"
+EnergyID Metric Kind:
+  description: "Data interpretation method - cumulative/total/delta/gauge (required)"
+Unit of Measurement:
+  description: "Units matching EnergyID's system (kWh, m³, °C, etc.) (required)"
+EnergyID Data Interval:
+  description: "Data resolution (P1D, PT1H, PT15M etc.) - defaults to daily (optional)"
+Upload Interval (seconds):
+  description: "Minimum time between uploads - defaults to 300s (5 mins) (optional)"
+{% endconfiguration_basic %}
+
+## Key features
+
+### Data handling
+
+- **Smart Upload Throttling**: Minimum 5-minute interval between updates
+- **Value Validation**: Automatic filtering of non-numeric states
+- **Connection Resilience**:
+  - 3 retry attempts with exponential backoff
+  - Automatic reconnection after network failures
+  - Configurable retry intervals (1s, 2s, 4s)
+
+### Metric types
+
+- **Cumulative**: For ever-increasing counters (e.g., total kWh)
+- **Total**: Interval-based consumption (e.g., daily usage)
+- **Delta**: Change between measurements
+- **Gauge**: Instantaneous readings (e.g., temperature)
+
+### Data resolution options
+
+| Interval | Description | Subscription required |
+|----------|-------------|-----------------------|
+| PT5M     | 5-minute    | Premium               |
+| PT15M    | 15-minute   | Premium               |
+| PT1H     | Hourly      | Free                  |
+| P1D      | Daily       | Free                  |
+
+## Installation guide
+
+### Obtaining a webhook URL
```

Comment:
should this section be part of the prereqs? It looks like you need this before you add the integration to Home Assistant?